### PR TITLE
Go over docs for Accuracy and Precision

### PR DESCRIPTION
**Pull request by rocky**
* Shy away from using markup that doesn't render properly in Django. In partuclar, math mode
* Add Wiki and WMA links in Accuracy and Precision builtin functions, and
* more cross refrences
* Move unit tests in autodoc to unit test
* reversed quote -> RawBackquote which is the WMA term for the character

<a href="https://gitpod.io/#https://github.com/Mathics3/mathics-core/pull/553"></a>

### CODE REVIEW

File: mathics/builtin/atomic/numbers.py [hunk 1]
```diff
@@ -954,40 +961,37 @@ def apply(self, expr, evaluation):
...
     <dl>
       <dt>'Precision[$expr$]'
       <dd>examines the number of significant digits of $expr$.
     </dl>
-    This is rather a proof-of-concept than a full implementation.
-    Precision of compound expression is not supported yet.
+
+    <i>This is rather a proof-of-concept than a full implementation.
+
+    Precision of compound expression is not supported yet.</i>
```
Comment (mmatera) [inline mathics/builtin/atomic/numbers.py:973]: Actually, with the last changes, compound expressions are also supported. I should have added some tests for that.

File: test/builtin/atomic/test_numbers.py [hunk 2]
```diff
@@ -177,6 +177,30 @@ def test_n():
...
+        ("1.0000000000000000 // Accuracy", "15."),
+        ("1.00000000000000000 // Accuracy", "17."),
+        # Returns the accuracy of ```2.4```
+        (" 0.4 + 2.4 I // Accuracy", "14.6198"),
+        ("Accuracy[2 + 3 I]", "Infinity"),
+        ('Accuracy["abc"]', "Infinity"),
+        # Returns the accuracy of ``` 3.2`3 ```
+        ('Accuracy[F["a", 2, 3.2`3]]', "2.49482"),
+        ('Accuracy[{{a, 2, 3.2`},{2.1`5, 3.2`3, "a"}}]', "2.49482"),
+        # Anoter case of issues with rounding. In Mathics, this returns
```
Comment (rocky) [inline test/builtin/atomic/test_numbers.py:197]: Anoter -> Another 

And thanks for adding all of the tests!

**Review by mmatera — APPROVED** [review]
I think now this is ready.

**Review by TiagoCavalcante — APPROVED** [review]
LGTM - just a small typo

File: mathics/doc/documentation/1-Manual.mdoc [hunk 3]
```diff
@@ -196,50 +196,46 @@ The result of the previous query to \Mathics can be accessed by '%':
...
+Above, two decimal places are shown in output after the decimal point, but three places of precision are stored.
 
-  >> a == 3.1413`4
+The relative uncertainty of '3.1416`3' is 10^-3. It is numerically equivalent, in three places after the decimal point, to 3.1413`4:
+
+  >> 3.1416`3 == 3.1413`4
    = True
 
-We can recover the precision of the number by using  'Precision' 
+We can get the precision of the number by using the \Mathics <url>:'Precision': /doc/reference-of-built-in-symbols/atomic-elements-of-expressions/representation-of-numbers/precision/</url> function:
```
[the remark this answers is not in the document]
Comment (mmatera) [reply mathics/doc/documentation/1-Manual.mdoc:211]: where?

[the remark this answers is not in the document]
Comment (TiagoCavalcante) [reply mathics/doc/documentation/1-Manual.mdoc:211]: It was fixed

**Comment by rocky** [comment]
@mmatera @TiagoCavalcante Any objections to merging this?

**Comment by mmatera** [comment]
> @mmatera @TiagoCavalcante Any objections to merging this?

Nop, go ahead!

**Comment by TiagoCavalcante** [comment]
> @mmatera @TiagoCavalcante Any objections to merging this? 

No, this is great!

**Pull request merged**